src/mame/barcrest/mpu4.cpp

```diff
@@ -2083,7 +2083,7 @@ void mpu4_state::mpu4_memmap(address_map &map)
 //  map(0x0800, 0x081f) // optional protection device lives here, see other maps
 	map(0x0850, 0x0850).rw(FUNC(mpu4_state::bankswitch_r), FUNC(mpu4_state::bankswitch_w));    /* write bank (rom page select) */
 	map(0x08e0, 0x08ef).rw(m_duart68681, FUNC(mc68681_device::read), FUNC(mc68681_device::write)); //Runs hoppers
-	map(0x08ed, 0x08ed).r(FUNC(mpu4_state::hack_duart_r)); // hack until the hopper is hooked up to the duart in games wanting that setup (eg m4ready)
+//	map(0x08ed, 0x08ed).r(FUNC(mpu4_state::hack_duart_r)); // hack until the hopper is hooked up to the duart in games wanting that setup (eg m4ready)
```

Comment:
If you remove it, you don't comment it, you just remove it.  We don't need remnants that way.

Reply:
Oops, missed that. Fixed